engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
@geoffjentry I think @gauravs90 is saying that the WA is also going to need a reference to the `MaterialieWorkflowDescriptorActor`. Does that mean we've decided that the WA will be responsible for asking `MaterialieWorkflowDescriptorActor` to create its own WorkflowDescriptor ?